src/OpenSSL/SSL.py

```diff
@@ -1531,7 +1563,7 @@ def set_app_data(self, data: Any) -> None:
         """
         self._app_data = data
 
-    def get_cert_store(self) -> X509Store:
+    def get_cert_store(self) -> Optional[X509Store]:
```

Comment:
This was a mistake in the type hints I submitted, I had not spotted the `None` case.